src/css/components/_control-bar.scss

```diff
@@ -29,9 +29,9 @@
   @include transition($trans);
 }
 
-.video-js.vjs-controls-disabled .vjs-control-bar,
-.video-js.vjs-using-native-controls .vjs-control-bar,
-.video-js.vjs-error .vjs-control-bar {
+.video-js.video-js.video-js.vjs-controls-disabled .vjs-control-bar,
+.video-js.video-js.video-js.vjs-using-native-controls .vjs-control-bar,
+.video-js.video-js.video-js.vjs-error .vjs-control-bar {
```

Comment:
this is necessarily because otherwise, the `.vjs-no-flex` selector below ends up being more specific than this and sets the control bar's display to table. This avoids using `!important` or really strong css selectors.